rust/src/lib/nm/bond.rs

```diff
@@ -46,9 +48,11 @@ fn apply_bond_options(
             .insert("ad_actor_sys_prio".to_string(), v.to_string());
     }
     if let Some(v) = bond_opts.ad_actor_system.as_ref() {
-        nm_bond_set
-            .options
-            .insert("ad_actor_system".to_string(), v.to_string());
+        if v != AD_ACTOR_SYSTEM_USE_BOND_MAC {
+            nm_bond_set
+                .options
+                .insert("ad_actor_system".to_string(), v.to_string());
+        }
```

Comment:
This is not true anymore. Please, see https://bugzilla.redhat.com/show_bug.cgi?id=1931881

Reply:
Thanks. Patch changed